spring-cloud-alibaba-docs/src/main/asciidoc-zh/seata.adoc

```diff
@@ -2,49 +2,71 @@
 
 === Seata 介绍
 
-Seata是一个分布式事务框架, 用于保证在微服务架构下每个服务的数据一致性
+Seata是一个分布式事务框架, 用于保证在微服务架构下每个服务的数据一致性。
 
 
 === 如何使用seata
 
 如果要在您的项目中引入 Seata，使用 group ID 为 `com.alibaba.cloud` 和 artifact ID 为 `spring-cloud-starter-alibaba-seata` 的 starter。
 
-```xml
+[source,xml]
+----
 <dependency>
     <groupId>com.alibaba.cloud</groupId>
     <artifactId>spring-cloud-starter-alibaba-seata</artifactId>
 </dependency>
-```
-
-在运行[spring-cloud-alibaba-examples/seata-example](https://github.com/alibaba/spring-cloud-alibaba/tree/2021.x/spring-cloud-alibaba-examples/seata-example)之前，您需要完成以下步骤：
-    - 1.保证 本地已经运行```nacos-server ```在 ```8848```端口
-    - 2.根据 ```spring-cloud-alibaba-examples/seata-example```下的 ```all.sql ```配置数据库
-    - 3.根据选择的事务模式，创建[事务日志表](https://github.com/seata/seata/tree/develop/script/client) 比如默认为AT模式，则进入at/db下选择对应的数据库脚本执行
-    - 4.创建 seata事务相关的[状态记录表](https://github.com/seata/seata/tree/develop/script/server/db) ```global_table```、```branch_table```、```lock_table```、```distributed_lock```
-    - 5.创建```spring-cloud-alibaba-examples/seata-example```所需的数据库表
-    - 6.启动```seata-server```
-
-    - 7.创建```spring-cloud-alibaba-examples/seata-example```中的Nacos配置，data id: seata.properties, Group: SEATA_ Group(Seata 1.5.1 默认组) 配置导入nacos配置在seata属性中添加示例中需要 的如下事务组配置
-    ```txt
-        service.vgroupMapping.order-service-tx-group=default
-        service.vgroupMapping.account-service-tx-group=default
-        service.vgroupMapping.business-service-tx-group=default
-        service.vgroupMapping.storage-service-tx-group=default
-    ```
-    - 8.根据seata官方提供的[seata-server.jar](https://seata.io/zh-cn/docs/ops/deploy-guide-beginner.html) 启动Seata Server
-    - 9.在本地启动 ```spring-cloud-alibaba-examples/seata-example```文件夹下的子服务  ``` business-service```, ```order-service ```, ```storage-service ```,最后启动全局事务控制服务 ``` account-service```
+----
+
+项目提供了相关实例 https://github.com/alibaba/spring-cloud-alibaba/tree/2021.x/spring-cloud-alibaba-examples/seata-example[Example]，在运行之前，您需要完成以下步骤：
```

Comment:
You should using https://github.com/alibaba/spring-cloud-alibaba/tree/2.2.x/spring-cloud-alibaba-examples/seata-example here.